spec/controllers/dashboard_controller_spec.rb

```diff
@@ -52,4 +54,33 @@
       end
     end
   end
+
+  describe "GET /kubectl-config" do
+    it "gets redirected if not logged in" do
+      get :kubectl_config
+      expect(response.status).to eq 302
+    end
+
+    context "HTML rendering" do
```

Comment:
We are actually setting `content_type: "text/yaml"` on the controller, it's the only mime type that we will return when asked for the kubectl config (at least for now). Moreover, we force the content disposition to be an attachment (so we force the browser to show the download dialog).